metadata-ingestion/docs/sources/mlflow/mlflow_pre.md

```diff
@@ -2,8 +2,10 @@
+|:-----------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------:|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|  [`Registered Model`](https://mlflow.org/docs/latest/model-registry/#registered-model)  |        [`MlModelGroup`](https://datahubproject.io/docs/generated/metamodel/entities/mlmodelgroup/)        | The name of a Model Group is the same as a Registered Model's name (e.g. my_mlflow_model). Registered Models serve as containers for multiple versions of the same model in MLflow.                                                                                                              |
+|   [`Model Version`](https://mlflow.org/docs/latest/model-registry/#model-version)    |             [`MlModel`](https://datahubproject.io/docs/generated/metamodel/entities/mlmodel/)             | The name of a Model is `{registered_model_name}{model_name_separator}{model_version}` (e.g. my_mlflow_model_1 for Registered Model named my_mlflow_model and Version 1, my_mlflow_model_2, etc.). Each Model Version represents a specific iteration of a model with its own artifacts and metadata. |
+|         [`Experiment`](https://mlflow.org/docs/latest/tracking/#experiments)         |           [`Container`](https://datahubproject.io/docs/generated/metamodel/entities/container/)           | Each Experiment in MLflow is mapped to a Container in DataHub. Experiments organize related runs and serve as logical groupings for model development iterations, allowing tracking of parameters, metrics, and artifacts.                                                                       |
+|            [`Run`](https://mlflow.org/docs/latest/tracking/#runs)             | [`DataProcessInstance`](https://datahubproject.io/docs/generated/metamodel/entities/dataprocessinstance/) | Each Run is mapped to a DataProcessInstance in DataHub, capturing the execution details, parameters, metrics, and lineage to a model.                                                                                                                                                            |
+|    [`Model Stage`](https://mlflow.org/docs/latest/model-registry/#deprecated-using-model-stages)     |                 [`Tag`](https://datahubproject.io/docs/generated/metamodel/entities/tag/)                 | The mapping between Model Stages and generated Tags is the following:<br/>- Production: mlflow_production<br/>- Staging: mlflow_staging<br/>- Archived: mlflow_archived<br/>- None: mlflow_none. Model Stages indicate the deployment status of each version.                         |
```

Comment:
this is actually deprecated in mlflow ([doc](https://mlflow.org/docs/latest/model-registry/#deprecated-using-model-stages)) -- maybe we should remove this? (and in the connector as well?)